core/mm/core_mmu.c

```diff
@@ -2418,10 +2418,19 @@ void *phys_to_virt(paddr_t pa, enum teecore_memtypes m, size_t len)
 		va = NULL;
 		break;
 	default:
+		if ( m == MEM_AREA_IO_SEC )
```

Comment:
Please remove the changes in this file, it looks like it's only debug code while you're testing things.

Reply:
I will do it when submitting this PR to the final review. Until then it will be in `draft state` and I will keep here some WIP or debug commits.